clients/fides-js/src/lib/consent-types.ts

```diff
@@ -154,6 +154,7 @@ export interface FidesGlobal extends Fides {
   shopify: typeof shopify;
   shouldShowExperience: () => boolean;
   showModal: () => void;
+  version?: string;
```

Comment:
I'd add this type to the documented `Fides` type instead. Assuming this works as intended, we'd want to include this in the dev docs.

Reply:
ah, missed that extend. good callout.